Fix: Formatting and Cleanup in Python Wrapper/API Files

AshiteshSingh opened the pull request:
Changes:
- Remove the extra semicolon after `("License");`

mihaimaruseac reviewed (changes requested):
There are 9 places in the codebase that have the same pattern. Please fix all in the same PR.

It is really a waste to have all CI run for just one character change

AshiteshSingh committed:
Fix formatting in TensorFlow wrapper and API files

Updated formatting in multiple Python wrapper files including
tfe_wrapper, c_api_experimental_reader, python_memory_checker_helper,
and pywrap_saved_model files.

AshiteshSingh commented:
I have fixed all other occurrences of the same extra semicolon across the repository as requested.

All changes are now included in this same PR.

Please review again. Thank you!

AshiteshSingh commented:
I have fixed the above problems and i am really sorry  by mistake those things occured and pls review again, thank you.

mihaimaruseac reviewed (changes requested):
Actually, no. You still have 12 commits and they are all "Update <file>".

12 commits for 9 characters.

AshiteshSingh commented:
Sorry, I accidentally re-requested review. The changes are complete - I squashed to 1 commit and removed the README changes as requested, Thank you

AshiteshSingh commented:
I have successfully change the title, you can review again,Thank you.
